packages/pagebuilder/lib/ContentTypes/Block/configAggregator.js

```diff
@@ -1,8 +1,16 @@
+import DOMPurify from 'dompurify';
```

Comment:
Multiple testcases are failing in this PR. Please look into them. Reason of failure could be third party module not found. Here is what we are getting in build failure.

'ERROR in ../pagebuilder/lib/ContentTypes/Block/configAggregator.js
--
566 | Module not found: Error: Can't resolve 'dompurify' in '/usr/src/app/packages/pagebuilder/lib/ContentTypes/Block'

Reply:
Hii @glo82145 
I got the Problem as package.json don't have dompurify dependencies install so added